src/worker.cc

```diff
@@ -971,23 +1085,16 @@ valhalla_exception_t::valhalla_exception_t(unsigned code, const std::string& ext
     message += ":" + extra;
 }
 
-void ParseApi(const std::string& request, Options::Action action, valhalla::Api& api) {
-  api.Clear();
-  auto document = from_string(request, valhalla_exception_t{100});
-  api.mutable_options()->set_action(action);
-  from_json(document, *api.mutable_options());
-}
-
-std::string jsonify_error(const valhalla_exception_t& exception, Api& request) {
+std::string serialize_error(const valhalla_exception_t& exception, Api& request) {
```

Comment:
i renamed this beacuse now the output is sometimes protobuf bytes rather than always json